src/app/pages/FastBtcPage/components/Deposit/MainScreen.tsx

```diff
@@ -69,11 +80,16 @@ export const MainScreen: React.FC<NetworkAwareComponentProps> = ({
             />
           )}
 
-          {fastBtcLocked && (
+          {(fastBtcReceiveLocked ||
```

Comment:
Export this condition to a new variable (e.g. `isInMaintenance`) and use it in `isDisabled` as well.